pkgs/development/tools/rust/cargo-tally/default.nix

```diff
@@ -2,14 +2,14 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-tally";
-  version = "1.0.18";
+  version = "1.0.19";
 
   src = fetchCrate {
     inherit pname version;
-    sha256 = "sha256-BlWPdZb85XaTGV6ZE3XRVKHJyXimfrezhRyqJVmCFMY=";
+    sha256 = "sha256-TlHb974Dtmz5qW+4L3BE6GQ3SbkwtIrZxE6FpJrgNdY=";
   };
 
-  cargoSha256 = "sha256-1qtlsItLP8MdxebgktzTr3R4Kq+PBIAiHGaikbQ796E=";
+  cargoSha256 = "sha256-np1rln9tD+J4s410/Pv4/eKAxyzK8qtLoON86JR1T94=";
 
   buildInputs = lib.optionals stdenv.isDarwin (with darwin.apple_sdk_11_0.frameworks; [
```

Comment:
Not related to this bump, but wouldn't this blow up on x86_64-darwin since you don't use the 11.0 SDK anywhere else (as in, you just use these frameworks from it)?

Reply:
the package doesn't seem to build with just `apple_sdk`, should I be overriding the stdenv then?

didn't do any further testing, but `testers.testVersion` succeeds on x86_64-darwin with `apple_sdk_11_0`

Reply:
@reckenrode Doesn't/shouldn't this not work? For context, the package is called with the standard callPackage, but somehow builds fine with the 11.0 frameworks.

Reply:
@winterqt It’s probably compiler-dependent. Clang and GCC need new stdenvs because they (as far as I can tell) hardcode some paths to the old SDK, and there’s no working around it otherwise. I’ve had similar problems with Rust in the past, but maybe that’s not the case anymore.

Reply:
Just making sure I merge this, this doesn't block this pr right?

Reply:
Nope.